WebApp/handler/ComponentHistoryHandler.py

```diff
@@ -0,0 +1,132 @@
+from WebAppDIRAC.Lib.WebHandler import WebHandler, WErr, WOK, asyncGen
+from DIRAC import gConfig, S_OK, S_ERROR, gLogger
+from DIRAC.Core.Utilities import Time
+from DIRAC.FrameworkSystem.Client.ComponentMonitoringClient import ComponentMonitoringClient
+import json
+import datetime
+
+class ComponentHistoryHandler( WebHandler ):
+
+  AUTH_PROPS = 'authenticated'
+
+  @asyncGen
+  def web_getInstallationData( self ):
+
+    req = self.__request()
+
+    client = ComponentMonitoringClient()
+    result = client.getInstallations( req[ 'installation' ], req[ 'component' ], req[ 'host' ], True )
+    if result[ 'OK' ]:
+      values = []
+      installations = result[ 'Value' ]
+      for i in range( self.pageNumber, self.pageNumber + self.numberOfInstallations ):
+        if len( installations ) > i:
+          installation = installations[ i ]
+        else:
+          break
+        uninstalled = '-'
+        if installation[ 'UnInstallationTime' ]:
+          uninstalled = installation[ 'UnInstallationTime' ].strftime( '%Y-%m-%d %H:%M' )
+        values.append( { 'Name': installation[ 'Instance' ], \
+                         'Module': installation[ 'Component' ][ 'Module' ], \
+                         'Host': installation[ 'Host' ][ 'HostName' ], \
+                         'System': installation[ 'Component' ][ 'System' ], \
+                         'Type': installation[ 'Component' ][ 'Type' ], \
+                         'Installed': installation[ 'InstallationTime' ].strftime( '%Y-%m-%d %H:%M' ), \
+                         'Uninstalled': uninstalled } )
+      timestamp = Time.dateTime().strftime( '%Y-%m-%d %H:%M [UTC]' )
+      total = len( installations )
+      callback = { 'success' : 'true', 'result' : values,
+                'total' : total, 'date' : timestamp }
+    else:
+      callback = { 'success' : 'false' , 'error' : result[ 'Message' ] }
```

Comment:
WErr.fromSERROR(result)
I know I also use this format, but I plan to replace to WErr